src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ApplicationSettingsControl.xaml

```diff
@@ -210,8 +210,16 @@
                     <ColumnDefinition Width="151"/>
                     <ColumnDefinition Width="50"/>
                 </Grid.ColumnDefinitions>
-                <Label Name="lblPlayScanningSoundLabel" Grid.Column="0" Content="{x:Static Properties:Resources.lblPlayScanningSoundLabelContent}" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
-                <ToggleButton Grid.Column="2" Style="{StaticResource tgbSlider}" Name="tgbtnPlayScanningSound" AutomationProperties.LabeledBy="{Binding ElementName=lblPlayScanningSoundLabel}" IsChecked="{Binding PlayScanningSound}"/>
+                <Label Name="lblSoundFeedbackLabel" Grid.Column="0" Content="{x:Static Properties:Resources.lblSoundFeedbackLabelContent}" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
```

Comment:
Line 211 of this file sets the control column width to 50, which was correct for the toggle but is too narrow for the combobox. We should set it to 180, which is the same width that is set for the highlight combobox.